docs/synthesis.md

```diff
@@ -146,12 +146,35 @@ Also, if present in the telemetry, these attributes are also added to the entity
 
 [guid_spec]: guid_spec.md
 
+#### Prefixed tags
+
+This is a special section where specify prefixes which are part of the attributes name.
```

Comment:
```suggestion
This is a special section where we specify prefixes which are part of the attribute name.
```

Reply:
Nit: A bit confusing this sentence to me, please double check

Reply:
Making it simpler as in the next paragraph starts the full explanation.